src/activities/number_click/Number_click.qml

```diff
@@ -0,0 +1,604 @@
+            Image {
+                id: tux
+                source: Activity.url + "tux.svg"
+
+                height: parent.height * 0.4
+                fillMode: Image.PreserveAspectFit
+
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.top: numericNumber.bottom
+                anchors.topMargin: parent.height * 0.05
+
+                MouseArea {
+                    id: tuxArea
+
+                    anchors.fill: parent
+                    hoverEnabled: true
+
+                    onEntered: {
+                        parent.rotation = -3
+                        parent.height = mainworkspace.height * 0.41
+                    }
+
+                    onExited: {
+                        parent.rotation = 0
+                        parent.height = mainworkspace.height * 0.40
+                    }
+
+                    onPressed: {
+                        parent.source = Activity.url + "tux_clicked.svg"
+                        activity.audioEffects.play("qrc:/gcompris/src/core/resource/sounds/smudge.wav")
+                        Activity.clickCount++
+                    }
+                    onReleased: { parent.source = Activity.url + "tux.svg" }
+
+
+                }
+            }
+
+            /*Button where the player has to click to check his answer*/
+            Rectangle {
+                id: buttonBox
+
+                height: parent.height * 0.12
+                width: parent.width * 0.7
+
+                color: "#00A31B"
+                border.width: 2
+                border.color: "grey"
+                radius: 60
+
+                anchors.horizontalCenter: parent.horizontalCenter
+                anchors.bottom: parent.bottom
+                anchors.bottomMargin: parent.height * 0.05
+
+                Image {
+                    id: buttonTick
+                    source: Activity.url + "tick.svg"
+
+                    height: parent.height * 0.6
+                    fillMode: Image.PreserveAspectFit
+
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    anchors.verticalCenter: parent.verticalCenter
+                }
+
+                MouseArea {
+                    id: buttonArea
+
+                    anchors.fill: parent
+                    hoverEnabled: true
+
+                    onEntered: { parent.color = "#00D022" }
+                    onExited: { parent.color = "#00A31B" }
+                    onPressed: {
+                        buttonTick.height = parent.height * 0.7
+                        activity.audioEffects.play("qrc:/gcompris/src/core/resource/sounds/bleep.wav")
+                        Activity.checkAnswer()
+                    }
+                    onReleased: { buttonTick.height = parent.height * 0.6 }
+                }
+            }
+        }
+
+        /* Set the error panel */
+        Rectangle {
+            id: errorpanel
```

Comment:
try to stick with the project naming convention, there is a mix between camelCase, snake_case and nocase